test/webpack-integration.test.js

```diff
@@ -47,24 +48,20 @@ describe('Webpack Integration Tests', () => {
           require(testFile)(suite);
         }
         const expectedDirectory = path.join(testDirectory, 'expected');
-        fs.readdirSync(expectedDirectory).forEach((file) => {
-          const filePath = path.join(expectedDirectory, file);
-          const actualPath = path.join(outputDirectory, file);
-          expect(readFileOrEmpty(actualPath)).toEqual(
-            readFileOrEmpty(filePath)
-          );
-          expect(readFileOrEmpty(actualPath)).toMatchSnapshot();
-        });
+
+        glob
```

Comment:
Why need glob here?

Reply:
I need to get all files in a directory recursively. I can do it without glob, but why not ?

Reply:
:+1: